src/App.scss

```diff
@@ -164,28 +164,47 @@ body {
     grid-column-end: 3;
     grid-row-start: 2;
     grid-row-end: 4;
-    display: grid;
+    display: flex;
+    flex-direction: column;
     grid-template-rows: min-content min-content auto;
 }
 
 .App__command-palette-command-container {
     background-color: #30444E;
     border-radius: 3px;
-    width: 100%;
     height: 100%;
+    width: 100%;
     overflow-y: auto;
     grid-row-start: 3;
     grid-row-end: 4;
     padding-top: 1rem;
 }
 
-.App__command-palette-commands {
-    width: 6rem;
+.App__command-palette-section-heading {
+    border-bottom: 1px solid #818A98;
+    color: #F1AE5B;
+    font-size: .9rem;
+    text-align: left;
+    padding-bottom: .6rem;
+    padding-left: 1.2rem;
+}
+
+.App__command-palette-section {
     display: grid;
+    grid-template-columns: 1fr 1fr;
+    padding: 0.4rem;
+
+    &:first-of-type {
+        box-shadow: 0px 3px 1px 0px #19282F;
+        margin-bottom: 0.5rem;
+    }
+}
+
+.App__command-palette-commands {
+    width: 3rem;
```

Comment:
There's some extra spaces to the right of the buttons when we set fixed width of 3rem, as each blocks have width of 2.5rem. That causes layout to be not horizontally centered; we can remove this width.